legal-api/src/legal_api/models/filing.py

```diff
@@ -133,7 +133,17 @@ class Source(Enum):
                 'ULC': 'BCINC',
                 'CC': 'BCINC',
                 'CP': 'OTINC'
-            }
+            },
+            'temporaryCorpTypeCode': 'TMP'
+        },
+        'registration': {
+            'name': 'registration',
+            'title': 'Registration',
+            'codes': {
+                'SP': 'FRREG',
+                'GP': 'FRREG'
+            },
+            'temporaryCorpTypeCode': 'RTMP'
```

Comment:
sorry, I'm missing why we need a temp type code as a business type.

Reply:
auth-web use this code to differentiate a draft IA or Registration. Until now we had only draft IA listing in my business registry dashboard, so when auth-web receives the code TMP it identifies as draft IA. I have an open pr to update auth-web according to this https://github.com/bcgov/sbc-auth/pull/1905